helm/hpcc/templates/dali.yaml

```diff
@@ -49,8 +49,11 @@ spec:
 {{- if hasKey $dali "labels" }}
 {{ toYaml $dali.labels | indent 8 }}
 {{- end }}
-{{- if hasKey $dali "annotations" }}
       annotations:
+{{- if hasKey $.Values.global "metrics" }}
```

Comment:
to avoid blank annotations: , perhaps this could be:
```
{{- if or (hasKey $dali "annotations") (hasKey $.Values.global "metrics") }}
      annotations:
```